artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyConnection.java

```diff
@@ -394,11 +393,11 @@ public String getRemoteAddress() {
 
    @Override
    public final String getLocalAddress() {
-      SocketAddress address = channel.localAddress();
+      String address = SocketAddressUtil.toString(channel.localAddress());
       if (address == null) {
          return null;
       }
-      return "tcp://" + IPV6Util.encloseHost(address.toString());
```

Comment:
This change could affect the users who filter by localAddress when listing connections, consumers, producers and sessions. Could you add a note for the next release in the versions.adoc file?